### ISSUE TITLE
Premium Expansion Support

### ISSUE THREAD

**Issue by Newsandbuy** [loki79uk/FS22_UniversalAutoload#215]
i couldnt find a discord so im posting it here: 

technically the pallets are supported, but the sizing is way of, so ive manually added the Pallets to my game, here is what im using

```xml
<containerConfiguration containerType="EURO_PALLET" name="pdlc_premiumExpansion:potatoChipsPallet" sizeX="1.25" sizeY="0.85" sizeZ="0.85"/>

<containerConfiguration containerType="EURO_PALLET" name="pdlc_premiumExpansion:vegetablesPallet" sizeX="1.65" sizeY="1.25" sizeZ="1.15"/>

<containerConfiguration containerType="EURO_PALLET" name="pdlc_premiumExpansion:preservedFoodParsnipPallet" sizeX="1.25" sizeY="0.7" sizeZ="0.85"/>

<containerConfiguration containerType="EURO_PALLET" name="pdlc_premiumExpansion:preservedFoodBeetRootPallet" sizeX="1.25" sizeY="0.7" sizeZ="0.85"/>

<containerConfiguration containerType="EURO_PALLET" name="pdlc_premiumExpansion:preservedFoodCarrotsPallet" sizeX="1.25" sizeY="0.7" sizeZ="0.85"/>
```

but i cant get `soupCansCarrots` `soupCansBeetRoot` or `soupCansParsnip` to work, also the `vegetablesPallet` stoped working at some point, it did work for an hour or so and without any change, the trailers wont recognize it anymore

**Comment by loki79uk** [comment]
Hi thanks for reminding me - I have been really busy with IRL work and forgot to even look at the new pallets in the expansion.  Are there any new trailers too?  I'll try to get this done today.

I actually have a new feature in development that will dynamically measure the size of any pallet, so it will do away with all of this config completely, but I need some more time to get it completed.

**Comment by Newsandbuy** [comment]
Dont worry, i just tried to figure it out myself and maybe help a bit with that,

yes there is a new `Farmtech DDK 2400` trailer wich has a bale configuration, its also the only new trailer.

**Comment by loki79uk** [comment]
I think this is everything added, I'll make a release now:

![image](https://github.com/loki79uk/FS22_UniversalAutoload/assets/76516343/0f7c7dff-1f9d-41c8-801e-6fe9e6bc0336)

**Comment by loki79uk** [comment]
Please try this one and let me know if you are still having any problems, or if there is anything missing!

https://github.com/loki79uk/FS22_UniversalAutoload/releases/tag/v1.4.3.0

**Comment by Newsandbuy** [comment]
That should be everything and it works with no issues on my end, thank you for your quick response.

**Comment by Newsandbuy** [comment]
sry i nearly forgot, there also is a new pickup truck, `Pickup 1984`

**Comment by loki79uk** [comment]
This seems prety good, will release again:
![image](https://github.com/loki79uk/FS22_UniversalAutoload/assets/76516343/98e61e11-72f9-4537-8cc4-66c749bcd2d3)

**Comment by Newsandbuy** [comment]
for some reason the profiliner will only load one row of soup can pallets, all other new pallets seem to work with no issue

![grafik](https://github.com/loki79uk/FS22_UniversalAutoload/assets/8875191/542a46ca-f331-46eb-9300-4adf9ff030de)

**Comment by loki79uk** [comment]
That is weird, they should be the same, but try making the soup pallet a fraction taller (in y)?

**Comment by Newsandbuy** [comment]
yes, changing the soup pallet from `sizeY="0.75"` to `sizeY="0.755"` fixes this issue

**Comment by NakataAkiyama** [comment]
> This seems prety good, will release again: ![image](https://user-images.githubusercontent.com/76516343/284069946-98e61e11-72f9-4537-8cc4-66c749bcd2d3.png)

Hello, when will the release with this pickup be released? And then only with a trailer, the latest release

**Comment by loki79uk** [comment]
> Hello, when will the release with this pickup be released? And then only with a trailer, the latest release

Sorry, I wasn't happy with it after all, but I've added it to
https://github.com/loki79uk/FS22_UniversalAutoload/releases/tag/v1.4.4.0

![image](https://github.com/loki79uk/FS22_UniversalAutoload/assets/76516343/4cc20e02-7014-401b-91fc-f081eefd2691)

**Comment by Newsandbuy** [comment]
i never new this mod supports the trains, but the new train doesnt work

**Comment by NakataAkiyama** [comment]
I hope the author will add support for the new train?

**Comment by loki79uk** [comment]
I will add the train soon.  I've been completely wiped out all week (since Monday night) with COVID. Still not back on my feet yet.. :(

**Comment by NakataAkiyama** [comment]
> I will add the train soon. I've been completely wiped out all week (since Monday night) with COVID. Still not back on my feet yet.. :(

Oh, get well soon, I hope your condition improves soon

**Comment by discord5000** [comment]
Re the first comment, there's definitely an issue with the mod sometimes not reading the modSettings\UniversalAutoload.xml file.
I'm trying to work out when it does and doesn't work.

**Comment by NakataAkiyama** [comment]
How are you? How are the successes of adding a train?
